src/OpenTelemetry.Contrib.Instrumentation.Quartz/OpenTelemetry.Contrib.Instrumentation.Quartz.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>OpenTelemetry Quartz.NET Instrumentation</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.Quartz-</MinVerTagPrefix>
+    <IncludeSharedInstrumentationSource>true</IncludeSharedInstrumentationSource>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```

Comment:
if Quartz supports, then consider targeting net461 as that's an officially supported .NET version.

Reply:
this has been updated